github-templates/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+(brief description / PR title)
+
+Status: **Ready for Review** or **Open for Visibility**
```

Comment:
+1 - this is the same one we've been using for most of our PR templates, unless there is a good reason to change it would be nice to have some continuity!